builtin/providers/aws/resource_aws_lb_cookie_stickiness_policy.go

```diff
@@ -41,7 +41,7 @@ func resourceAwsLBCookieStickinessPolicy() *schema.Resource {
 
 			"cookie_expiration_period": &schema.Schema{
 				Type:     schema.TypeInt,
-				Optional: true,
+				Required: true,
```

Comment:
Instead of making this required, I think we can keep it optional, adding `Default: 0` should solve the problem with _DuplicatePolicyName_ you mentioned.